packages/accordion/src/AccordionItem.ts

```diff
@@ -22,6 +22,7 @@ import { Focusable } from '@spectrum-web-components/shared/src/focusable.js';
 import { when } from '@spectrum-web-components/base/src/directives.js';
 import '@spectrum-web-components/icons-ui/icons/sp-icon-chevron100.js';
 import chevronIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-chevron.css.js';
+import chevronIconOverrides from '@spectrum-web-components/icon/src/icon-chevron-overrides.css.js';
 
```

Comment:
@nikkimk Should we be importing the overrides in the components like this or should we keep this task tightly coupled to the css files only as we do for other component overrides? 
For ex. in `accordion` we import the overrides in `accordion.css` file instead of adding those in the `Accordion.ts` file.
for icons we don't have custom css files and that's why I understand you've added these overrides in the `.ts` files itself. But this would affect the performance if I understand correctly as we are importing multiple css files in the component. Instead in the earlier case since we were importing only one css file for each component this was a better performant system. What are your thoughts on this?

Reply:
Let's not remove anything in this PR, better we should create a parallel PRs for removals.